src/components/molecules/InternalModalTransition/InternalModalTransition.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC } from "react";
```

Comment:
Extrapolated out as Sonarcloud wasn't happy and code for the most part is re-useable